crates/store/Cargo.toml

```diff
@@ -16,6 +16,7 @@ deadpool-sqlite = { version = "0.8", features = ["rt_tokio_1"] }
 directories = { version = "5.0" }
 figment = { version = "0.10", features = ["toml", "env"] }
 hex = { version = "0.4" }
+miden-crypto = { version = "0.11", default-features = false }
```

Comment:
Related to the last comment, we probably don't need to introduce this dependency.